pymc3/plots/pairplot.py

```diff
@@ -0,0 +1,162 @@
+import warnings
+import matplotlib.gridspec as gridspec
+
+try:
+    import matplotlib.pyplot as plt
+except ImportError:  # mpl is optional
+    pass
+from ..util import get_default_varnames, is_transformed_name, get_untransformed_name
+from .artists import get_trace_dict, scale_text
+
+
+def pairplot(trace, varnames=None, figsize=None, text_size=None, 
+                gs=None, ax=None, hexbin=False, plot_transformed=False, 
+                divergences=False, kwargs_divergence=None, 
+                sub_varnames=None, **kwargs):
+
+    """
+    Plot a scatter or hexbin matrix of the sampled parameters.
+
+    Parameters
+    ----------
+
+    trace : result of MCMC run
+    varnames : list of variable names
+        Variables to be plotted, if None all variable are plotted
+    figsize : figure size tuple
+        If None, size is (8 + numvars, 8 + numvars)
+    text_size: int
+        Text size for labels
+    gs : Grid spec
+        Matplotlib Grid spec.
+    ax: axes
```

Comment:
Passing `ax` and `gs` could be confusing, could we just use `gs`? I see the `ax` is there for the special case of plotting a two variable pairplot, but the same effect can be achieved using just `gs`,  right?